packages/ckeditor5-link/src/linkimageediting.js

```diff
@@ -28,7 +30,7 @@ export default class LinkImageEditing extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ ImageEditing, LinkEditing ];
+		return [ ImageEditing, ImageBlock, ImageInline, LinkEditing ];
```

Comment:
This would disable an option to select only `ImageBlock` or `ImageInline`.

Reply:
Done.